.github/workflows/eof-validation.yml

```diff
@@ -0,0 +1,12 @@
+name: Validate EOF presence
+
+on:
+  push:
+    branches:
+      - test-ruleset-branch
+  workflow_dispatch:
+
+jobs:
+  eof-validation:
+    uses: ai-cfia/github-workflows/.github/workflows/workflow-verification-eof.yml@23-create-workflow-to-verify-eof-line-presence
```

Comment:
we should probably point to a more generic workflow that can validate more than just EOF (naming conventions for example). perhaps ```workflow-file-standards-validation```

Reply:
yeah true makes sense